### ISSUE TITLE
Radio field doesn't allow for "other" choice

### ISSUE THREAD

**Issue by DevinWalker** [Desertsnowman/Caldera-Forms#319]
Regularly I create radio field lists with the option to include an "Other" option with a text field:

![11-3-2015 10-59-44 pm](https://cloud.githubusercontent.com/assets/1571635/10931724/e7409730-827e-11e5-9252-0daa5481cc1d.png)

It would be awesome if Caldera could include this functionality:

![11-3-2015 10-59-44 pm](https://cloud.githubusercontent.com/assets/1571635/10931732/f18e5d94-827e-11e5-9d53-88959755514b.png)

**Comment by DevinWalker** [comment]
Notice the "Enabled other choice" checkbox

**Comment by Desertsnowman** [comment]
you could just add an "Other" option then add a text field bellow the list and set a condition to show if option is "Other".

**Comment by Shelob9** [comment]
Yes to what David says, but would still be an interesting feature. The two field solution means two fields in CSV output....

**Comment by DevinWalker** [comment]
@Shelob9 yeah it would be nice to have them in the same field, report, export
